app/lib/frontend/templates/views/security.mustache

```diff
@@ -0,0 +1,18 @@
+{{! Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+<div style="max-width: 600px">
+  <h2>Security</h2>
+  <p>
+      The Dart team takes the security of Pub and its published packages seriously.
```

Comment:
I think we should be more general here: s/of Pub and its published packages/of Pub.dev/